### PR TITLE
Add iOS/Android native libs

### LINKED ISSUE #11: Add iOS Support
Bootstrap [bugsplat-ios](https://github.com/BugSplat-Git/BugSplat-iOS) via the bugsplat-unreal plugin.

### LINKED ISSUE #12: Add Android Support
Bootstrap [crashpad](https://github.com/chromium/crashpad) via the bugsplat-unreal plugin.

### PR DESCRIPTION
**Pull request by tustanivsky**
Fixes #12 Fixes #11

### CODE REVIEW

File: Source/BugSplatRuntime/Private/BugSplatRuntime.cpp [hunk 1]
```diff
@@ -0,0 +1,132 @@
+// Copyright Epic Games, Inc. All Rights Reserved.
```
Comment (bobbyg603) [inline Source/BugSplatRuntime/Private/BugSplatRuntime.cpp:1]: Should this be Copyright BugSplat?

File: Source/BugSplatRuntime/Private/BugSplatEditorSettings.cpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#include "BugSplatEditorSettings.h"
```
Comment (bobbyg603) [inline Source/BugSplatRuntime/Private/BugSplatEditorSettings.cpp:1]: Does this need Copyright info?

File: Source/BugSplatRuntime/Private/BugSplatRuntime.cpp (hunk 1 above)
Comment (bobbyg603) [inline Source/BugSplatRuntime/Private/BugSplatRuntime.cpp:1]: Should this be Copyright BugSplat?

File: Source/Scripts/setup-upload-symbols.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+export bugSplatUser=$(awk -F "=" '/BugSplatUser/ {print $2}' ${configPath}/DefaultEngine.ini)
+export bugSplatPassword=$(awk -F "=" '/BugSplatPassword/ {print $2}' ${configPath}/DefaultEngine.ini)
+
+if [ -z "$bugSplatUser" ]; then
+    echo "BugSplat postprocessing: bugSplatUser variable is empty"
+fi
+
+if [ -z "$bugSplatPassword" ]; then
+    echo "BugSplat postprocessing: bugSplbugSplatPassword variable is empty"
```
Comment (bobbyg603) [inline Source/Scripts/setup-upload-symbols.sh:57]:
```suggestion
    echo "BugSplat postprocessing: bugSplatPassword variable is empty"
```

File: Source/Scripts/setup-upload-symbols.sh [hunk 4]
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```
Comment (bobbyg603) [inline Source/Scripts/setup-upload-symbols.sh:1]: Very well written script, good variable names, easy to follow, good logging. Thanks!

File: Source/Scripts/upload-symbols.sh [hunk 5]
```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```
Comment (bobbyg603) [inline Source/Scripts/upload-symbols.sh:1]: TODO BG, extract this to a submodule?

**Comment by bobbyg603** [comment]
Initial review looks pretty good. Will make a full QA pass over the next few days.

File: Source/Scripts/upload-symbols.sh (hunk 5 above)
Comment (tustanivsky) [reply Source/Scripts/upload-symbols.sh:1]: Here `upload-symbols.sh` script is different from the one that is provided along with the native iOS SDK, so I'm not sure whether it's worth extracting it to a submodule.

**Comment by bobbyg603** [comment]
Great work! Looks like there's work to do on our end to replace the Crashpad libraries but I think we can follow up with that work.

**Pull request merged**